### ISSUE TITLE
[Datamart] Get RMA list

### ISSUE THREAD

**Issue by hoatv214** [tikivn/open-api-docs#197]
Need to get RMA list for return process.

**Comment by vietcaominh** [comment]
@tuhucon Please help check it

**Comment by vietcaominh** [comment]
hi @hoatv214 You want to get RMA orders?

**Comment by hoatv214** [comment]
Yes, we do. We have a feature, Return Reconciliation, to have sellers to verify whether their return parcels are actually back to their warehouse or not. Now, we dont know which orders will be returning in advance. That's why we need a RMA list, or maybe you can have a better idea to crack this problem?

Here is the original discussion:

![image](https://user-images.githubusercontent.com/53849748/83712474-19e11f80-a650-11ea-958f-6608604d8c09.png)
![image](https://user-images.githubusercontent.com/53849748/83712502-26fe0e80-a650-11ea-88fd-4dd3fc37844f.png)

**Comment by vietcaominh** [comment]
ok, @hoatv214 
Let us create a backlog for it!

**Comment by TrongNguyen3103** [comment]
Hi, 
I need to get list rma order via API, what should I do?
Tks
